core/src/main/resources/error/error-classes.json

```diff
@@ -757,6 +757,16 @@
       "The expression <sqlExpr> cannot be used as a grouping expression because its data type <dataType> is not an orderable data type."
     ]
   },
+  "HLL_INVALID_LOG_K" : {
+    "message" : [
+      "Invalid call to <function>; the log K value must be between <min> and <max>, inclusive: <value>."
+    ]
+  },
+  "HLL_UNION_DIFFERENT_LOG_K" : {
+    "message" : [
+      "Sketches have different lgConfigK values: <left> and <right>. Set the allowDifferentLgConfigK parameter to true to use the <function> of different lgConfigK values."
```

Comment:
Here you are using `lgConfigK`. I feel this is better.
Also (style): why not put `allowDifferentLgConfigK` inside single quotes to stress that it is an actual parameter (copied here verbatim)? Same fir `true`. Also instead of "of different", maybe "with different"?
See below for the complete proposal:
```
Sketches have different lgConfigK values: <left> and <right>. Set the `allowDifferentLgConfigK` parameter to `true` to call <function> with different lgConfigK values.
```

Reply:
Sounds good, done.